frontend/src/lib/lemon-ui/Link/Link.tsx

```diff
@@ -183,6 +188,16 @@ export const Link: React.FC<LinkProps & React.RefAttributes<HTMLElement>> = Reac
                         router.actions.push(to)
                     }
                 }
+            } else {
+                if (openInNewPostHogTab) {
+                    // For internal links, open in new PostHog tab
+                    event.preventDefault()
+                    event.stopPropagation()
+                    if (to && typeof to === 'string') {
+                        newTab(to)
```

Comment:
I suggest something like this instead:
```suggestion
                        sceneLogic.isMounted() ? sceneLogic.actions.newTab(to) : window.open(to)
```